docs/ocs-endpoint-v2.yaml

```diff
@@ -0,0 +1,202 @@
+          type: string
+        subjectRichParameters:
+          description: (Optional) Subject parameters for `subjectRich` (see [Rich Object String](https://github.com/nextcloud/server/issues/1706))
+          type: array
+          items:
+            $ref: "#/components/schemas/RichStringParameter"
+        message:
+          description: (Optional) Translated potentially longer message that should be presented to the user
+          type: string
+        messageRich:
+          description: (Optional) Translated message string with placeholders (see [Rich Object String](https://github.com/nextcloud/server/issues/1706))
+          type: string
+        messageRichParameters:
+          description: (Optional) Message parameters for `messageRich` (see [Rich Object String](https://github.com/nextcloud/server/issues/1706))
+          type: array
+          items:
+            $ref: "#/components/schemas/RichStringParameter"
+        link:
+          description: (Optional) A link that should be followed when the subject/message is clicked
+          type: string
+          format: url
+        icon:
+          description: (Optional) A link to an icon that should be shown next to the notification.
+          type: string
+          format: url
+        actions:
+          description: (Optional) An array of action elements
+          type: array
+          items:
+            $ref: "#/components/schemas/Action"
+    Action:
+      type: object
+      required:
+        - label
+        - link
+        - type
+        - primary
+      properties:
+        label:
+          description: Translated short label of the action/button that should be presented to the user
+          type: string
+        link:
+          description: A link that should be followed when the action is performed/clicked
+          type: string
+        type:
+          description: HTTP method that should be used for the request against the link
+          type: string
+          enum: [GET, POST, DELETE]
+        primary:
+          description: If the action is the primary action for the notification or not
+          type: boolean
+    Notifications:
+      type: object
+      required:
+        - ocs
+      properties:
+        ocs:
+          type: object
+          properties:
+            meta:
+              $ref: "#/components/schemas/Meta"
+            data:
+              type: array
+              items:
+                $ref: "#/components/schemas/Notification"
+    RichStringParameter:
+      type: object
+      required:
+        - type
+        - id
+        - name
+      properties:
+        type:
+          description: A unique identifier for the object type
+          type: string
+        id:
+          description: A short identifier of the object on the server (int or string)
+          type: string
+        name:
+          description: A name which should be used in the visual representation
+          type: string
+    Meta:
```

Comment:
The order feels a bit weird, should we just "fix it" to the logical order?
Notifications > Meta > Notifcation > Rich... > Action

Reply:
Could be done, or maybe alphabetical or more convenient.